test/functional/vimscript/system_spec.lua

```diff
@@ -660,4 +660,33 @@ describe('shell :!', function()
         1]])
     end
   end)
+
+  it(':{range}! with powershell without redirecting to buffer', function()
```

Comment:
Should we perhaps remove the "with powershell" part from the test text if we're testing non-powershell as well?

Reply:
Yes.  Done.